process/block/block_processor_test.go

```diff
@@ -0,0 +1,282 @@
+package block_test
+
+import (
+	"encoding/json"
```

Comment:
sort imports

Reply:
Did not have import auto-format installed.
It is changed now